jsk_mbzirc_common/package.xml

```diff
@@ -17,6 +17,16 @@
 
   <build_depend>gazebo_ros</build_depend>
   <build_depend>std_msgs</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+
+  <run_depend>geometry_msgs</run_depend>
+  <run_depend>hector_quadrotor</run_depend>
+  <run_depend>hector_localization</run_depend>
+  <run_depend>hector_models</run_depend>
+  <run_depend>hector_gazebo</run_depend>
+  <run_depend>hector_slam</run_depend>
+  <run_depend>hector_quadrotor_demo</run_depend>
+
```

Comment:
@k-okada 
it seem like only need to install " hector_quadrotor_demo",
other packages would install automatically.